protocols/generic_ngs_prep/README.md

```diff
@@ -0,0 +1,110 @@
+# Custom NGS Library Prep
+
+### Author
+[Opentrons](https://opentrons.com/)
+
+## Categories
+* NGS Library Prep
+
+## Description
+This protocol performs a generic NGS Library Prep. It utilizes up to three temperature modules in Slot 1, 4 and 7 for the cooling of reagents. It also utilizes the magnetic module for bead based purification of samples.
+
+Explanation of complex parameters below:
+* `P20 Multichannel GEN2 Mount Position`: Choose the mount position of the P20 Multichannel GEN2 pipette.
+* `P300 Multichannel GEN2 Mount Position`: Choose the mount position of the P300 Multichannel GEN2 pipette.
+* `Number of Samples`: Enter total number of samples in the protocol run. **Note: Because both pipettes are 8-channels, the number of samples should be a multiple of 8 or close if possible, otherwise it will use up additional tips for an entire column.**
+
```

Comment:
You should include a description of all of the parameters (that are in fields.json) here.

Reply:
bump

Reply:
Done